pkg/reconciler/pipelinerun/affinity_assistant.go

```diff
@@ -207,6 +244,8 @@ func affinityAssistantStatefulSet(name string, pr *v1beta1.PipelineRun, claimNam
 			Selector: &metav1.LabelSelector{
 				MatchLabels: getStatefulSetLabels(pr, name),
 			},
+			// by setting VolumeClaimTemplates from StatefulSet, all the PVs are scheduled to the same Availability Zone as the StatefulSet
+			VolumeClaimTemplates: claimTemplates,
```

Comment:
Since PVCs created by statefulset volumeClaimTemplates are not deleted when the statefulset is deleted, we'll need to update existing cleanup logic to delete these PVCs.

Reply:
Updated the cleanup logic to delete PVCs!